packages/pilot/src/components/MetricList/index.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import shortid from 'shortid'
+import {
+  Card,
+  CardTitle,
+  CardContent,
+} from 'former-kit'
+
+import MetricListItem, { listItemPropTypes } from './MetricListItem'
+
+import styles from './style.css'
+
+const MetricList = ({
+  items,
+  title,
+}) => (
+  <Card>
+    <CardTitle title={title} />
+    <CardContent
+      className={styles.list}
+    >
+      {items.map(({
+        icon,
+        title: itemTitle,
+        value,
+      }) => (
+        <MetricListItem
+          key={shortid.generate()}
```

Comment:
aqui acho interessante utilizar como key index e value pois dentro do nosso contexto conseguimos garantir que o componente vai ser reaproveitado em re-render, usando shortid o componente sempre vai ser renderizado novamente
```suggestion
          key={`${index}${value}`}
```

Reply:
Corrigido!